app/config/projects.yml

```diff
@@ -0,0 +1,279 @@
+parameters:
+    doctrine.projects:
+        orm:
+            name: Object Relational Mapper
+            shortName: ORM
+            slug: orm
+            docsSlug: doctrine-orm
+            composerPackageName: doctrine/orm
+            repositoryName: doctrine2
+            docsRepositoryName: doctrine2
+            docsPath: /docs
+            description: PHP object relational mapper (ORM) that sits on top of a powerful database abstraction layer (DBAL). One of its key features is the option to write database queries in a proprietary object oriented SQL dialect called Doctrine Query Language (DQL). This provides developers with a powerful alternative to SQL that maintains flexibility without requiring unnecessary code duplication.
```

Comment:
Consider using [yaml multiline syntax here](https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines#21699210)